search/src/main/java/artic/edu/search/SearchResultsManager.kt

```diff
@@ -119,22 +111,103 @@ class SearchResultsManager(private val searchService: SearchServiceProvider,
         }
     }
 
-    private fun loadArtwork(artwork: List<ApiSearchContent.SearchedArtwork>?): Observable<List<ArticObject>> {
-        return if (artwork == null) {
-            Observable.just(emptyList())
-        } else {
-            articObjectDao.getObjectsByIdList(artwork.map { it.artworkId.toString() }).toObservable()
+    private fun loadArtwork(artwork: List<ApiSearchContent.SearchedArtwork>?): Observable<List<ArticSearchArtworkObject>> {
+        return articDataObjectDao.getDataObject()
+                //Get the base Server url just in case we need it... which we might
+                .map { dataObject -> dataObject.imageServerUrl }
+                .toObservable()
+                .flatMap {
+                    generateArtworkObjectList(it, artwork)
+                }
+
+    }
+
+    private fun generateArtworkObjectList(baseUrl: String,
+                                          artwork: List<ApiSearchContent.SearchedArtwork>?)
```

Comment:
`artwork` -> `artworks`

Reply:
Unsure of the logic of this (artwork and artworks both sound fine) but done.